drive/client.bal

```diff
@@ -305,16 +305,16 @@ public client class Client {
     # 
     # + fileId - ID of the file to be updated
     # + optional - 'UpdateFileMetadataOptional' used to add query parameters to the request
-    # + fileResource - 'File' can added as a payload to change metadata
+    # + fileMetadata - 'FileMetadata' can added as a payload to change metadata
     # + return - If successful, returnsoptionalsFileMetadata `File`. Else returns `error`
     @display {label: "Update File Metadata By Id"}
     remote isolated function updateFileMetadataById(@display {label: "File Id"} string fileId, 
                                                     @display {label: "File Resource"} 
```

Comment:
New File Matadata?

Reply:
I did not see create file operation in the diff. Please check that also. File metadata should be an optional input.

Reply:
it was removed in the last commit 
https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/commit/07ae38137a5b20a9a77a1da33d539699966bd575